data/themes/darktable.css

```diff
@@ -1841,6 +1841,15 @@ Details :
   background-color: @lighttable_bg_color;
 }
 
+/* here we set the maximum size of GUI elements in thumbtable view to be in synch with the rest of the GUI */
+#thumbtable_filemanager,
+#thumbtable_filmstrip,
+#thumbtable_zoom,
+#culling
+{
+min-height: 1.47em;
```

Comment:
I'm not fan of abusing css names like that... But maybe that's the only solution...

Reply:
Me too. Hope a better solution could be possible.